navigation.py

```diff
@@ -137,9 +138,10 @@ def find_button(driver, button_id=None, class_name=None, xpath=None):
         class_name -- a CSS class of the button to be returned. If multiple form fields share the same class, then the first field that has the class is used.
         xpath -- XPath of the element to be returned.
 
-    It is compulsory to specify precisely one of button_id and class_name, otherwise the function will raise an InvalidArgumentsError"""
-    if not ((button_id is not None) ^ (class_name is not None) ^ (xpath is not None)):
-        raise InvalidArgumentError("Precisely one of button_id, class_name, or xpath has to be specified.")
+    It is compulsory to specify at most one of button_id and class_name, otherwise the function will raise an InvalidArgumentError.
```

Comment:
Also include `xpath` in this documentation.

Reply:
It also sounds more natural to say "No more than one of ... can be specified, otherwise ..."